orbit/pkg/user/user_darwin.go

```diff
@@ -0,0 +1,30 @@
+//go:build darwin
+// +build darwin
+
+package user
```

Comment:
Yeah, I'm not a big fan of our multiple narrow-scoped packages under `orbit/pkg` as that tends to only accentuate the issue - there's no sensible place to add some new stuff so we create more small packages, etc. But that's no big deal for now, we can address that eventually and try to consolidate some of those.